### DISCUSSION TITLE
Why symlink all that stuff to the support directory?

### DISCUSSION THREAD

**Discussion by MikeLund** [downforcetech/fluidxml.php#19] [Questions]
Just wondering why you do this? I dislike using symlinks, and I don't see the benefit of them in your repository. I also don't like dotfiles that are unneeded, especially in a dependency.

Maybe I misunderstand something? Thanks :)

**Comment by daniele-orlando** [comment]
Hi dear Mike.
I prefer to not use symlinks when possible too, but sometimes they can help.

They are there for mainly two reasons:
1) (my needs) I code with Vim and Command-T and the latter doesn't like a lot files having a dot at the beginning of their name. So to reach them easily I need these files to be named without the dot at the beginning.
2) (others' needs) Dot files are by nature hidden files and not every OS makes easy to show hidden files. Some people for example use MacOS which hides dot files and prevents from opening them with double click. I know know, you can change the defaults to force Finder showing them, but not everyone apply this trick to MacOS.

Said that, we came to the point that having files without the dot at the beginning helps me and people in general to deal with configuration files. That's why I symlinked dot files to their corresponding file without the dot.
Why under the 'support' directory? To avoid messing the root of the project with duplicated files.

Clarified the mystery.
